src/hockeypuck/go.mod

```diff
@@ -14,11 +14,12 @@ require (
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/lib/pq v1.10.9
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.17.0
+	github.com/prometheus/client_golang v1.14.0
```

Comment:
Why downgrading?

Reply:
Good catch, I didn't see this. No idea where this came from, this definitely isn't intentional. Will chase up before merging, smells fishy.

Reply:
Ugh, this appears to be a difference of opinion between Martin's local golang install and mine. He did a lot of dependency housekeeping in da608f7, and for some reason my local golang/vscode decided to fight back, hence the dependency churn in subsequent commits, particularly 9e245e5 and c077cfd. It's also quite possible that my aggressive rebasing has made matters worse by reordering commits. Vendoring makes quite a mess of the git history... :-(

I'll bump this back up by hand for now, and try to clean up the dependency tree at a later date.